BrainPortal/app/models/cluster_task.rb

```diff
@@ -1542,11 +1542,10 @@ def unarchive_work_directory
   # is the task's results_data_provider_id.
   # Restoring the state of the workdir can be performed
   # with unarchive_work_directory_from_userfile().
-  def archive_work_directory_to_userfile(dp_id = nil)
-    return false unless self.archive_work_directory
+  def archive_work_directory_to_userfile(dp_id = nil, nozip = nil)
+    return false unless self.archive_work_directory(nozip)
     file_id  = self.workdir_archive_userfile_id
     return true if file_id
-
     full=self.full_cluster_workdir
```

Comment:
Why remove the blank line? I like blank lines that properly separate small pieces of code. Small paragraphs that do things related to each other.

Reply:
Also please return the blank line

Reply:
rolled back